dapps/assets/src/components/DataContainer.ts

```diff
@@ -80,6 +80,13 @@ export default class DataContainerComponent extends mixins(EvanComponent) {
     this.hasError = false;
     this.searchTerm = searchTerm;
 
+    // Necessary for proper routing. Otherwise collision with detail dapp
```

Comment:
where does this come from now? I have seen it before, but it doesn't seem ticket related?